Client/TabManagement/Tab.swift

```diff
@@ -958,9 +958,13 @@ class TabWebView: WKWebView, MenuHelperInterface {
         self.accessoryView = AccessoryViewProvider()
 
         super.init(frame: frame, configuration: configuration)
+        accessoryView.previousClosure = {
```

Comment:
Should we check CC feature flag status before setting the closure to the tab?

Reply:
Good question, although the arrows should work regardless of CC as they are for all input fields.

Reply:
However this accessory view should only show up for when we have credit card enabled so if that is not happening then we have an issue ha

For this case you can leave it the way it is :)